examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java

```diff
@@ -211,19 +178,61 @@ public static void main(String[] args) throws IOException {
     PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.CountWords());
 
     /**
-     * Concept #5: Format the results for a BigQuery table, then write to BigQuery.
-     * The BigQuery output source supports both bounded and unbounded data.
+     * Concept #5: Key by the window, so we can write one file per window. To access the window in a
```

Comment:
I did not initially want to add a concept, but it is pretty easy and otherwise the APIs for `DoFn` extra params which are, sadly, less discoverable than autocompleted methods.